eg/eg_exec.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 import argparse
-import eg_config
-import eg_util
+from eg import eg_config
+from eg import eg_util
```

Comment:
Is this `from eg import ...` relevant to python3, or just a stylistic thing? I want to keep the ability to run `eg_exec.py` as a standalone script, without pip installation, via something like `python eg_exec.py find`, and this `from` syntax seems to cause those invocations to fail.